tdvt/tdvt/tdvt.py

```diff
@@ -415,6 +415,21 @@ def get_level_of_parallelization(args):
         run-pattern postgres_odbc --logp logicaltests/setup/calcs/setup.BUGS.*.dbo.xml --tdp cast_calcs.*.tds --test-ex 59740
 
     '''
+run_connectors_test_usage_text = '''
```

Comment:
This may be a big nit, but I wonder if this command should start with `test` instead of `run`. The other `run` commands are for the TDVT tool itself.

Reply:
I had used the name run_connectors_test because we are running the connectors test using this command. Also i may be misunderstanding the second statement, but isnt this command also for the tdvt tool ?

Reply:
Sorry, my misunderstanding. Since this is testing the connector, not the TDVT tool, `run` is the correct start :)

Reply:
Ran the args test which take care of the change because the change is basically the parsing of args